plugins/gcp/resoto_plugin_gcp/resources/base.py

```diff
@@ -61,7 +61,7 @@ def do_work() -> T:
             self.futures.append(future)
         return future
 
-    def wait_for_submitted_work(self) -> None:
+    def _wait_for_current_batch(self) -> None:
```

Comment:
move the function inside wait_for_submitted_work. Nobody should call this method ever.